tutorials/connect-jamf-pro-to-smallstep.mdx

```diff
@@ -0,0 +1,200 @@
+
+### Connect Jamf to Smallstep
+
+Let’s add the Jamf API credentials you just created to Smallstep.
+
+1. In the Smallstep UI, go to the [**Device Management**](https://smallstep.com/app/?next=/settings/devices) tab in ⛭ **Settings**
+2. Under Jamf, choose ➕ **Connect**
+3. Enter the API credentials from Jamf
+4. Choose **Add Platform**. Your device inventory will start syncing from Jamf to Smallstep.
+5. Temporarily save the **Webhook URL**, **Username**, and **Password** presented.
+
+### Configure a SCEP Enrollment webhook in Jamf Pro
+
+For compatibility reasons, Smallstep may use ACME or SCEP for certificate enrollment. The SCEP Enrollment webhook allows Jamf to request a dynamic, short-lived challenge string from Smallstep during a SCEP request. The challenge string is injected into a [SCEP payload](https://developer.apple.com/documentation/devicemanagement/scep) for your clients to authenticate to Smallstep.
+
+1. In Jamf Pro, choose ⚙️ **Settings**
+2. Under the **Global** tab, Choose **Webhooks**
+3. Add a new Webhook
+    1. Display name: Smallstep
+    2. Webhook URL: (details from step #2)
+    3. Authentication Type: Basic
+        1. Basic Authentication Username: (details from step #2)
+        2. Basic Authentication Password: (details from step #2)
+        3. Basic Authentication Verify Password: (details from step #2)
+    4. Content Type: JSON
+    5. Webhook Event: SCEPChallenge
+4. Choose **Save**
+
+### Create an Agent Package for Distribution
+
+In this step, you’ll upload the Smallstep agent package to Jamf’s distribution network.
+
+1. Download the latest package from [packages.smallstep.com](https://packages.smallstep.com/stable/darwin/step-agent-plugin_latest.pkg)
+2. In Jamf Pro, choose ⚙️ **Settings**
+3. Under the **Computer Management** tab, Choose **Packages**
+4. Add a new Package
+    1. Display name: Smallstep Agent
+    2. Filename: (upload from step #1)
+5. Choose **Save**
+
+### Create an Agent Bootstrap Script
+
+This step will install a script on your client devices that bootstraps the connection between your devices and Smallstep.
+
+1. In Jamf Pro, choose ⚙️ **Settings**
+2. Under the **Computer Management** tab, Choose **Scripts**
+3. Add a new Script
+    1. In the **General tab**, for **Display Name**: `Smallstep Agent Install`
+    2. In the Script tab:
+        - Mode: `Shell/Bash`
+        - Content:
+            
+            ```console
+            launchctl stop com.smallstep.launchd.Agent
+            /Applications/SmallstepAgent.app/Contents/MacOS/SmallstepAgent svc install
+            ```
+            
+4. Choose **Save**
+
+### Create an Agent Installation Policy
+
+Next, we’ll configure the Script we just created to run on your client devices.
+
+1. In Jamf Pro, choose 💻 **Computers**
+2. Under the **Content Management** tab, choose **Policies**
+3. Add a new **Policy**
+    1. Under Options → General:
+        - Display name: Smallstep Agent
+        - Trigger: Login
```

Comment:
@joshdrake is Login still the correct trigger? Or should this be set to `Recurring Check-in`?

Reply:
Ok I'll add a PR to update it